### PR TITLE
[Spark-Core][Test][Minor]: Fix the wrong comment in test

### PR DESCRIPTION
**Pull request by wangmiao1981**
## What changes were proposed in this pull request?

While learning core scheduler code, I found two lines of wrong comments. This PR simply corrects the comments.

### CODE REVIEW

File: core/src/test/scala/org/apache/spark/scheduler/TaskSetManagerSuite.scala [hunk 1]
```diff
@@ -265,10 +265,10 @@ class TaskSetManagerSuite extends SparkFunSuite with LocalSparkContext with Logg
     // get chosen before the noPref task
     assert(manager.resourceOffer("exec1", "host1", NODE_LOCAL).get.index == 2)
 
-    // Offer host2, exec3 again, at NODE_LOCAL level: we should choose task 2
+    // Offer host2, exec2, at NODE_LOCAL level: we should choose task 3
```
Comment (srowen) [inline core/src/test/scala/org/apache/spark/scheduler/TaskSetManagerSuite.scala]: Is this comment correct? the assertion compares to 1, not 3. I might misunderstand this.

Comment (wangmiao1981) [inline core/src/test/scala/org/apache/spark/scheduler/TaskSetManagerSuite.scala]: The resource offering is host = `host2` and executor = `exec2`. So the comment should reflect this offering. 

The `index` is actually the index of tasks in the FakeTaskSet. Let me double check whether I should change this line or the previous example.

Comment (wangmiao1981) [inline core/src/test/scala/org/apache/spark/scheduler/TaskSetManagerSuite.scala]: Tasks in TaskSet:
0 List(executor_host1_exec1)
1 List(executor_host2_exec2)
2 List(host1, executor_host2_exec2)
3 List()
I will make changes accordingly.

**Comment by srowen** [comment]
Merged to master

**Pull request closed without merging**